ynput/OpenPype
# Nuke: fixing thumbnail and monitor out root attributes

## Changelog Description
Nuke Root Colorspace settings for Thumbnail and Monitor Out schema was gradually changed between version 12, 13, 14 and we needed to address those changes individually for particular version.

## Testing notes:
1. Open Settings at `project_settings/nuke/imageio/workfile/monitorLut`
2. Set the value to either `ACES/sRGB` or `sRGB (ACES)` but make sure the OCIO config is set to ACES config.
3. Open Nuke with 12, 13, 14 minor versions and use Set Colorspace menu item .
4. Look to Script properties panel to Colorspace Tab and see that the Thumbnail, Monitor Out attributes were set correctly

- [ ] Nuke 12.x 
- [ ] Nuke 13.x
- [ ] Nuke 14.x

### warning for testing Nuke 12
Once you will be testing Nuke 12 and you will open workfile which was originally created at Nuke 13/14 this will popup and it is not an isssue of this PR. This will disapeare as soon as you save the script in 12 and then reopen again.

![image](https://github.com/ynput/OpenPype/assets/40640033/dec2a459-05f7-45b6-bf71-c3d4bb0c3e37)
